packages/cms/package.json

```diff
@@ -11,7 +11,9 @@
     "release": "canon-release",
     "setup": "canon-setup",
     "start": "canon-start",
-    "migrate-db": "node scripts/migrate-db.js"
+    "migrate-legacy": "node scripts/migration/migrate-legacy.js",
+    "migrate-0.1": "node scripts/migration/migrate-0.1.js",
+    "migrate-0.6": "node scripts/migration/migrate-0.6.js"
```

Comment:
_cracks knuckles_ 🤓 

Let's get these scripts exported from this package. Adding these to the `package.json` "scripts" object works locally when in this folder, but do not "ship" with the package. For that, we need to add them to the "bin" object. Check out how `canon-core` is setup: https://github.com/Datawheel/canon/blob/master/package.json#L6-L12

Additionally, let's give the commands a little more namespace. How about `canon-cms-migrate-*`?

Aaaaaand finally, as an FYI, projects can run a package's script by typing `npx script-name` instead of having to actually add it to the package.json "scripts" object, which is great for one-time scripts like these (so that's what we should use in usage documentation).

Reply:
@jhmullen and ANOTHER thing: at some point ago, to get the "bin" scripts picked up for local testing, it was necessary to also do the following (YMMV):
* Add [this one-liner](https://github.com/Datawheel/canon/blob/master/bin/dev.js#L1) to the top of any main script file: `#! /usr/bin/env node`
* After you add the scripts to package.json, you needed to run `npm run link` to "register" them in your local env. This script gets run automatically when installing the package in another repo.

Reply:
@davelandry added readme with `npx`, and namespaced the scripts!